deploy/templates/service-template.yml

```diff
@@ -0,0 +1,330 @@
+---
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: assisted-migration-service
+parameters:
+  # Migration Planner Service
+  - name: MIGRATION_PLANNER_IMAGE
+    required: true
+  - name: MIGRATION_PLANNER_IMAGE_TAG
+    value: latest
+  - name: MIGRATION_PLANNER_REPLICAS
+    description: Number of replicas of the service to run.
+    value: "3"
+  - name: MIGRATION_PLANNER_MEMORY_REQUEST
+    description: Memory request for the API pods.
+    value: "400Mi"
+  - name: MIGRATION_PLANNER_MEMORY_LIMIT
+    description: Memory limit for the API pods.
+    value: "2000Mi"
+  - name: MIGRATION_PLANNER_CPU_REQUEST
+    description: CPU request for the API pods.
+    value: "300m"
+  - name: MIGRATION_PLANNER_CPU_LIMIT
+    description: CPU limit for the API pods.
+    value: "500m"
+  - name: MIGRATION_PLANNER_AGENT_IMAGE
+    description: Migration Discovery Service image
+    required: true
+  - name: MIGRATION_PLANNER_AGENT_URL
+    description: The service URL valid to the service from the agent
+    required: true
+  - name: MIGRATION_PLANNER_UI_URL
+    description: The UI URL valid to the service from the agent
+    required: true
+  
+  # Migration Planner UI
+  - name: MIGRATION_PLANNER_UI_IMAGE
+    required: true
+  - name: MIGRATION_PLANNER_UI_IMAGE_TAG
+    value: latest
+  
+  # Postgres
```

Comment:
We need to be prepared for AWS RDS and not lock on PVC, would those variables work also for that?